example/ck_tile/01_fmha/codegen/ops/fmha_fwd.py

```diff
@@ -494,6 +494,20 @@ def get_pipelines(dtype, hdim) -> List[FmhaFwdPipeline]:
                     cond &= pipeline.F_squant == 'f'
                     if not cond:
                         continue
+                if receipt == 3:
```

Comment:
```suggestion
                if receipt in (3, 4):
```
would avoid some repetition to achieve the same effect

Reply:
looks like 3 & 4 are different. but 3 is same as 2 (3 is redundant but cannot remove, maybe someone use that...)